account-kit/smart-contracts/src/ma-v2/account/semiModularAccountV2.ts

```diff
@@ -0,0 +1,151 @@
+import type {
+  EntryPointDef,
+  SmartAccountSigner,
+  SmartContractAccountWithSigner,
+  ToSmartContractAccountParams,
+} from "@aa-sdk/core";
+import {
+  createBundlerClient,
+  getEntryPoint,
+  toSmartContractAccount,
+} from "@aa-sdk/core";
+import {
+  concatHex,
+  encodeFunctionData,
+  getContract,
+  maxUint32,
+  toHex,
+  type Address,
+  type Chain,
+  type Hex,
+  type Transport,
+  hexToBigInt,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'hexToBigInt' is defined but never used.
<!-- __reviewdog__:ChA3ZjEwMDkwZGNhY2M3ZWJhEgZyZGpzb24= -->